src/Data/Neuroglia.Data/Properties/LocalizedResources.fr.resx

```diff
@@ -189,6 +189,9 @@
   <data name="exception_domain_argumentcannotbeadateinthefuture" xml:space="preserve">
     <value>L'argument '{argument}' ne peut pas être une date future</value>
   </data>
+  <data name="exception_domain_argumentcannotbeadateinthepast" xml:space="preserve">
+    <value>L'argument '{argument}' ne peut pas être une date passée</value>
```

Comment:
> L'argument '{argument}' ne peut pas être une date passée

Maybe best to say:

`L'argument '{argument}' ne peut pas être une date dans le passé`

@JBBianchi WDYT?

Reply:
I think too, I write it to have the same wording as `exception_domain_argumentcannotbeadateinthefuture`.